src/main/java/step4/model/LottoWinningStatistics.java

```diff
@@ -1,44 +1,44 @@
 package step4.model;
 
-import step3.exception.LottoFormatException;
-import step4.constant.ErrorMessageConstant;
-
 import java.util.*;
+import java.util.stream.Collectors;
 
 public class LottoWinningStatistics {
-    private final Map<Rank, Integer> lottoWinningStatistics = new LinkedHashMap<>();
+    private static final Map<Rank, Integer> lottoWinningStatistics;
+
+    static {
+        lottoWinningStatistics = initLottoWinningStatistics();
+    }
```

Comment:
불필요한 static 블럭은 사용하지 않으셔도 될 것 같아요!!😆

```suggestion
    private static final Map<Rank, Integer> lottoWinningStatistics = initLottoWinningStatistics();
```